pyCFOFiSAX/_cfofisax.py

```diff
@@ -100,19 +100,19 @@ def init_forest_isax(self, size_word: int, threshold: int,
     def score_icfof(self, query: np_array, ntss: np_ndarray, rho=[0.001, 0.005, 0.01, 0.05, 0.1],
                     each_tree_score: bool = False, fast_method: bool = True):
         """
-        Calcul les approximations *i*\ CFOF.
-        Appelle une des deux fonctions selon le paramètre ``fast_method`` :
-         - si ``True`` (par défaut) : :func:`~pyCFOFiSAX._forest_iSAX.ForestISAX.vranglist_by_idtree_faster`
-         - si ``False`` : :func:`~pyCFOFiSAX._forest_iSAX.ForestISAX.vranglist_by_idtree`
-        Puis trie la liste des vrang pour obtenir les approximations CFOF en fonction des valeurs de ``rho`` en paramètre.
-
-        :param numpy.array query: la séquence à évaluer
-        :param numpy.ndarray ntss: les séquences de réference
-        :param list rho: les valeurs de rho pour le calcul des approximation
-        :param bool each_tree_score: si vrai, retourne les scores obtenus dans chacun des arbres
-        :param bool fast_method: si vrai, utilise les fonctions numpy pour le calcul, sinon parcourt l'arbre via une liste FIFO de nœuds
-
-        :returns: les approximations *i*\ CFOF
+        Calculate *i*\ CFOF Approximations.
+        Call one of the two functions according to the parameter ``fast_method`` :
+         - if ``True`` (default) : :func:`~pyCFOFiSAX._forest_iSAX.ForestISAX.vranglist_by_idtree_faster`
+         - if ``False`` : :func:`~pyCFOFiSAX._forest_iSAX.ForestISAX.vranglist_by_idtree`
+        Then sort the vrang list to get CFOF approximations based on ``rho`` parameter values.
+
+        :param numpy.array query: The sequence to be evaluated
+        :param numpy.ndarray ntss: Reference sequences
+        :param list rho: Rho values for calculating approximation
+        :param bool each_tree_score: if vrai, teturns the scores obtained in each of the trees
+        :param bool fast_method: if vrai, uses the NUMPY functions for the calculation, otherwise travels the tree via a FIFO list of nodes
```

Comment:
```suggestion
        :param bool fast_method: if `True`, uses the numpy functions for computation, otherwise goes  through the tree via a FIFO list of nodes
```